streaming/index.js

```diff
@@ -733,27 +732,30 @@ const startServer = async () => {
           }
 
           // Check filters
-          if (req.cachedFilters && !unpackedPayload.filtered) {
-            const status = unpackedPayload;
+          if (req.cachedFilters && !payload.filtered) {
+            const mutatedPayload = { ...payload };
+            const status = payload;
             const searchContent = ([status.spoiler_text || '', status.content].concat((status.poll && status.poll.options) ? status.poll.options.map(option => option.title) : [])).concat(status.media_attachments.map(att => att.description)).join('\n\n').replace(/<br\s*\/?>/g, '\n').replace(/<\/p><p>/g, '\n\n');
             const searchIndex = JSDOM.fragment(searchContent).textContent;
 
             const now = new Date();
-            payload.filtered = [];
+            mutatedPayload.filtered = [];
```

Comment:
You could avoid defining `mutatedPayload`, have `const filtered = []` (or a more explicit name) here, and then below use use `transmit({ ...payload, filtered })`?

Reply:
Also did this in my refactoring PR, but used a reduce instead of array declaration / forEach / push